requirements.txt

```diff
@@ -0,0 +1,14 @@
+--editable git+git://github.com/dr3y/dnaplotlib.git@master#egg=dnaplotlib
+bokeh
+matplotlib
+networkx
+fa2
+lxml
+beautifulsoup4
+cython
+sympy
+pandas
+astropy
+emcee
+python-libsbml
+#--editable git+git://github.com/biocircuits/bioscrape.git@master#egg=bioscrape
```

Comment:
do you want to remove this for now?

Reply:
ok let me remove it and make sure the binder still works. it should though

Reply:
i tried to make it load bioscrape obviously that didnt work

Reply:
I figured, but let's not accumulate dead code.